recipes/matplotlib-scalebar/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "matplotlib-scalebar" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "4b398f3d11c2d1ec4baa9e92212665ff58ec4ae2a2e0d92a87b80e750dcc4b14"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - matplotlib
+    - pip
+    - python
+  run:
+    - matplotlib
+    - python
+
+test:
+  imports:
+    - matplotlib_scalebar
+
+about:
+  home: "https://github.com/ppinard/matplotlib-scalebar"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE"
+  summary: "Artist for matplotlib to display a scale bar"
+  doc_url: "https://github.com/ppinard/matplotlib-scalebar"
+  dev_url: "https://github.com/ppinard/matplotlib-scalebar"
+
+extra:
+  recipe-maintainers:
+    - ppinard
```

Comment:
@ppinard: are you happy with being a maintainer of this recipe?

Reply:
Sure. Thank you for creating the recipe.